pkg/util/k8sutil/tls.go

```diff
@@ -16,6 +16,7 @@ package k8sutil
 
 import (
 	"github.com/coreos/etcd-operator/pkg/util/etcdutil"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Small nit: I'm pretty sure this isn't aligned correctly. Check out the ordering around line 21